source/form.html

```diff
@@ -5,7 +5,157 @@
+              <label class="label label--checkbox" for="souvenirs">Накупил сувениров</label>
+            </li>
+            <li class="contest-feedback__item">
+              <input class="checkbox" type="checkbox" name="tower" id="tower" checked>
+              <label class="label label--checkbox" for="tower">Удержал башню</label>
+            </li>
+            <li class="contest-feedback__item">
+              <input class="checkbox" type="checkbox" name="hotel" id="hotel">
+              <label class="label label--checkbox" for="hotel">Разгромил отель</label>
+            </li>
+           </ul>
+        </fieldset>
+      </div>
+      <div class="contest-feedback__wrapper">
+        <fieldset class="contest-feedback__field contest-feedback__field--contacts">
+          <ul class="contest-feedback__list">
+            <li class="contest-feedback__item">
+              <label class="label label--caption" for="phone">Номер телефона</label>
+              <input class="tel" type="tel" name="phone" id="phone" placeholder="Номер, пожалуйста">
+            </li>
+            <li class="contest-feedback__item">
+              <label class="label label--caption" for="email">Адрес почты</label>
+              <input class="text" type="text" name="email" id="email" placeholder="Введите почту" required>
+            </li>
+          </ul>
+        </fieldset>
+      </div>
+      <div class="contest-feedback__wrapper">
+        <fieldset class="contest-feedback__field contest-feedback__field--app">
+          <legend class="contest-feedback__title">С каким приложением путешествовали?</legend>
+          <ul class="contest-feedback__list">
+            <li class="contest-feedback__item">
+              <input class="radio" type="radio" name="ios" id="ios" checked>
+              <label class="label label--radio" for="ios">Pink для iOS</label>
+            </li>
+            <li class="contest-feedback__item">
+              <input class="radio" type="radio" name="android" id="android">
+              <label class="label label--radio" for="android">Pink на Android</label>
+            </li>
+            <li class="contest-feedback__item">
+              <input class="radio" type="radio" name="win-phone" id="win-phone">
+              <label class="label label--radio" for="win-phone">Windows Phone</label>
+            </li>
+          </ul>
+        </fieldset>
+      </div>
+      <div class="contest-feedback__wrapper">
+        <fieldset class="contest-feedback__field contest-feedback__field--emotion">
+          <legend class="contest-feedback__title">Опишите свои эмоции</legend>
+          <textarea class="textarea-text" name="feedback-emotion" id="feedback-emotion" placeholder="Можно прям в красках, не стесняясь в выражениях"></textarea>
+        </fieldset>
+      </div>
+      <div class="contest-feedback__wrapper contest-feedback__wrapper--submit">
+        <input class="contest-feedback__submit" type="submit" value="Отправить форму">
```

Comment:
Почему не `button`?
```suggestion
        <button class="contest-feedback__submit" type="submit">Отправить форму</button>
```
Это более современный синтаксис.